lib/utils/testLauncherHelper.js

```diff
@@ -54,23 +54,14 @@ function promptPassword() {
 }
 
 /**
- * Read `.suitestrc` or `.suitestrc.js` launcher config file.
+ * Read `.suitestrc` launcher config file.
  * If file not found, return empty object.
+ * Supports json and ini formats.
+ * cli arguments are not parsed.
  * If file found, but json ivalid, throw error.
  */
 function readLauncherConfig() {
-	// result.config is the parsed configuration object
-	// result.filepath is the path to the config file that was found
-	const result = cosmiconfig('suitest', {
-		rc: '.suitestrc',
-		js: 'suitestrc.js',
-		packageProp: false,
-		rcExtensions: true,
-		rcStrictJson: true,
-		sync: true,
-	}).load();
-
-	return result ? result.config : {};
+	return rc('suitest', {}, () => ({}));
```

Comment:
We should consider actually providing arguments parser (3rd argument) and let RC lib do all the props merging. But that can wait till next release.